From 5fc166b05611bd5cf7db83284181416e1c08a350 Mon Sep 17 00:00:00 2001
From: Leonardo Arena <rnalrd@alpinelinux.org>
Date: Wed, 19 Dec 2018 10:11:38 +0000
Subject: [PATCH] fix build on ppc64le

/home/buildozer/aports/community/openzwave/src/openzwave-1.4.164/cpp/src/command_classes/DoorLockLogging.cpp:312:15: error: passing argument 1 to restrict-qualified parameter aliases with argument 4 [-Werror=restrict]

Use default Alpine optimization flag
---
 cpp/build/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cpp/build/Makefile b/cpp/build/Makefile
index a632f79..2d7c337 100644
--- a/cpp/build/Makefile
+++ b/cpp/build/Makefile
@@ -16,7 +16,7 @@
 DEBUG_CFLAGS    := -Wformat -Wall -Wno-unknown-pragmas -Wno-inline -Werror -Wno-error=sequence-point -Wno-sequence-point -ggdb -DDEBUG -fPIC  -DSYSCONFDIR="\"$(PREFIX)/etc/openzwave/\""
 DEBUG_CPPFLAGS  := -std=c++11 
 # what flags we will use for compiling in release mode
-RELEASE_CFLAGS  := -Wall -Wno-unknown-pragmas -Werror -Wno-error=sequence-point -Wno-sequence-point -O3 -DNDEBUG -fPIC  -DSYSCONFDIR="\"$(PREFIX)/etc/openzwave/\""
+RELEASE_CFLAGS  := -Wall -Wno-unknown-pragmas -Werror -Wno-error=sequence-point -Wno-error=restrict -Wno-sequence-point -Os -DNDEBUG -fPIC  -DSYSCONFDIR="\"$(PREFIX)/etc/openzwave/\""
 RELEASE_CPPFLAGS := -std=c++11
 #what flags we will use for linking in debug mode
 DEBUG_LDFLAGS	:= -g
